sdk/spring/CHANGELOG.md

```diff
@@ -62,7 +63,7 @@ This section includes changes in `spring-cloud-azure-autoconfigure` module.
 
 #### Bugs Fixed
 - Fix the issue that `AzureMessagingListenerAutoConfiguration` not included in `spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports` [#34690](https://github.com/Azure/azure-sdk-for-java/issues/34690), [#35717](https://github.com/Azure/azure-sdk-for-java/pull/35717).
-- Improve default AAD configuration condition [#36126](https://github.com/Azure/azure-sdk-for-java/pull/36126).
+- Improve default Microsoft Entra configuration condition [#36126](https://github.com/Azure/azure-sdk-for-java/pull/36126).
```

Comment:
Needn't change the old changelog, what do you think @saragluna ?

Reply:
As discussed, keep the old name in the history logs.